insta_reels_publishing_api_sample/index.js

```diff
@@ -102,8 +102,9 @@ app.post("/uploadReels", async function (req, res) {
         const igResponse = await axios.get(getInstagramAccountUri);
         const hasIgBusinessAccount = igResponse.data.instagram_business_account ? true : false;
         const igUserId = igResponse.data.instagram_business_account.id;
-        const { videoUrl, caption } = req.body;
-        const uploadVideoUri = `https://graph.facebook.com/v14.0/${igUserId}/media?media_type=REELS&video_url=${videoUrl}&caption=${caption}&access_token=${req.session.userToken}`;
+        const { videoUrl, caption, coverUrl, thumbOffset } = req.body;
+        const uploadParamsString = `caption=${caption}&cover_url=${coverUrl}&thumb_offset=${thumbOffset}&access_token=${req.session.userToken}`;
```

Comment:
If you provide both offset and url, only url will be applied. Can we make sure user only provides one of these options in the UI?

Reply:
done